CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
 - [#35](https://github.com/pusher/oauth2_proxy/pull/35) arm and arm64 binary releases (@kskewes)
   - Add armv6 and arm64 to Makefile `release` target
 - [#37](https://github.com/pusher/oauth2_proxy/pull/37) cross build arm and arm64 docker images (@kskewes)
+- [#68](https://github.com/pusher/oauth2_proxy/pull/68) forward X-Auth-Access-Token header (@davidholsgrove)
```

Comment:
This isn't in quite the right place, needs to go into the changes since v3.1.0 section please

```
## Changes since v3.1.0

<-- here -->

# v3.1.0
``

Reply:
Whoops! Yes sorry about that - moved it now :D